app/routables/index/styles.scss

```diff
@@ -1,168 +1,307 @@
+        top: 35px;
+        right: 95px;
+        z-index: 2;
+        cursor: pointer;
+        width: 50px;
+        height: 50px;
+        opacity: 0;
       }
-    }
-  }
-
-  section#hero {
-    background-color: #252525;
-    background-image: url("/assets/images/jc-bag.jpg");
-    height: 95vh;
-    background-position: center;
-    background-repeat: no-repeat;
-    background-size: cover;
-    position: relative;
-  }
-
-  section#products {
-    background-color: #1c1c1c;
-    display: flex;
-    padding: 70px 40px;
-    ul.column {
-      display: flex;
-      flex-flow: wrap;
-      justify-content: center;
-      width: 1440px;
-      li.product {
-        max-width: 346px;
-        padding: 0px 20px;
-        .image {
-          position: relative;
-          img {
-            width: 100%;
-            height: auto;
-          }
-          // price tag
-          p {
-            font-family: $bold;
-            position: absolute;
-            background: #ffdb5b;
-            color: #353535;
-            top: 20px;
-            right: 20px;
-            padding: 10px 16px;
-            font-size: 22px;
-            margin: 0;
-          }
+      .toggler:checked + .hamburger {
+        div {
+          transform: rotate(135deg);
         }
-        .text {
-          padding: 28px;
-          position: relative;
-          background-color: #272727;
-          margin-bottom: 40px;
-          h4 {
-            font-family: $bold;
-            font-size: 30px;
-            color: white;
-            small {
-              font-family: $light;
-              font-size: 24px;
-              margin-left: 10px;
-            }
-          }
-          p,
-          dt,
-          dd {
-            color: white;
-            line-height: 180%;
-            font-size: 16px;
-          }
-          dl {
-            margin: 22px 0;
-            display: flex;
-            flex-flow: wrap;
-            dt {
-              width: 40%;
-            }
-            dd {
-              width: 60%;
-              margin: 0; // to overide the browser default.
-            }
-          }
-          p {
-            a {
-              color: white;
-              text-decoration: underline;
-              &:hover {
-                text-decoration: none;
-                color: #ffdb5b;
-              }
-            }
+        div::before,
+        div::after {
+          top: 0;
+          transform: rotate(90deg);
+        }
+        .toggler:checked ~ .menu {
+          visibility: visible;
+          div {
+            transform: scale(1);
+            transition-duration: 0.8s;
           }
         }
       }
-    }
-  }
-}
+      .menu {
+        position: fixed;
+        top: 0px;
+        right: 0px;
+        width: 100%;
+        height: 100%;
+        visibility: hidden;
+        overflow: hidden;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+      .one {
+        background: #040009;
```

Comment:
Consider creating SASS variables so that readers know what colour this is.
```
$dark-red: #801617;
$sky-blue: #040009;
```
```
.one {
  background: $sky-blue;
}
```